### PR TITLE
Add Lepidopter release blog post, fix link

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

**Comment by anadahz** [comment]
cc: @agrabeli

File: content/post/lepidopter.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: "OONI releases Lepidopter Raspberry Pi distribution"
+author: "Vasilis Ververis"
+date: "2016-07-04"
+tags: ["release", "raspberry pi", "distribution", "image", "lepidopter"]
+categories: ["blog"]
+---
+
+The Open Observatory of Network Interference (OONI) is happy to announce a new
+distribution image dedicated to embedded devices such as Raspberry Pi named
```
Comment (agrabeli) [inline content/post/lepidopter.md]: I would rephrase this to: "The Open Observatory of Network Interference (OONI) team is pleased to announce the release of an ooniprobe distribution for embedded devices, such as Raspberry Pis, called "Lepidopter".

File: content/post/lepidopter.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+## What is Lepidopter
+
+Lepidopter is a Raspberry Pi distribution image with all the required
+dependencies and software packages in place, configured to run individual
+networks measurements tests via the
+[ooni-probe](https://github.com/TheTorProject/ooni-probe) software.  It is
+developed and designed to require no physical attendance upon first bootstrap
+but also allows experienced users to further configure it as they wish. The
+[source code](https://github.com/TheTorProject/Lepidopter), and the building
+scripts of Lepidopter image are under a free/libre source software; GPL version
```
Comment (agrabeli) [inline content/post/lepidopter.md]: I would rephrase this to: "The source code and the building scripts of the Lepidopter image are free and open source software."

File: content/post/lepidopter.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+3 license which can be reviewed and modified by anyone.
+
+## Why to use it
+
+Starting from 2012 OONI has collected an amount of network measurements reports
```
Comment (agrabeli) [inline content/post/lepidopter.md]: Replace "Starting from" with "Since".

File: content/post/lepidopter.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+networks measurements tests via the
+[ooni-probe](https://github.com/TheTorProject/ooni-probe) software.  It is
+developed and designed to require no physical attendance upon first bootstrap
+but also allows experienced users to further configure it as they wish. The
+[source code](https://github.com/TheTorProject/Lepidopter), and the building
+scripts of Lepidopter image are under a free/libre source software; GPL version
+3 license which can be reviewed and modified by anyone.
+
+## Why to use it
+
```
Comment (agrabeli) [inline content/post/lepidopter.md]: Replace "an amount" with "numerous" (or the actual amount).

File: content/post/lepidopter.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+Starting from 2012 OONI has collected an amount of network measurements reports
+that show evidence of internet censorship, network tampering and surveillance.
+OONI measurement data have been used in a big amount of research publications,
```
Comment (agrabeli) [inline content/post/lepidopter.md]: Include a reference/link for "research publications".

Comment (agrabeli) [inline content/post/lepidopter.md]: I'm not sure OONI measurement data has been used in a large/big amount of research publications. I would just say: "OONI measurement data has been used in research publications".

File: content/post/lepidopter.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+a number of technical about how ISP in countries filter and censor the internet
+as well as reports that include sing of network irregularities and tampering.
```
Comment (agrabeli) [inline content/post/lepidopter.md]: Replace "sing" with "signs".

File: content/post/lepidopter.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+[source code](https://github.com/TheTorProject/Lepidopter), and the building
+scripts of Lepidopter image are under a free/libre source software; GPL version
+3 license which can be reviewed and modified by anyone.
+
+## Why to use it
+
+Starting from 2012 OONI has collected an amount of network measurements reports
+that show evidence of internet censorship, network tampering and surveillance.
+OONI measurement data have been used in a big amount of research publications,
+a number of technical about how ISP in countries filter and censor the internet
```
Comment (agrabeli) [inline content/post/lepidopter.md]: Add the word "reports" after technical? It would be good to add a reference/link here as well.

Comment (agrabeli) [inline content/post/lepidopter.md]: Replace "ISP" with "ISPs".

File: content/post/lepidopter.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+as well as reports that include sing of network irregularities and tampering.
+This wouldn't be possible without the help of numerous volunteers that are
+contributed by running consistent ooni-probe network measurements tests from
+different vantage points. Up to now mostly technical savvy users needed to
+maintain their own probes and deal with system configuration, software upgrades
+and renewal of measurement test and relevant test content related to the
```
Comment (agrabeli) [inline content/post/lepidopter.md]: Perhaps replace this with "upgrading to the latest version of ooniprobe"?

File: content/post/lepidopter.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+dependencies and software packages in place, configured to run individual
+networks measurements tests via the
+[ooni-probe](https://github.com/TheTorProject/ooni-probe) software.  It is
+developed and designed to require no physical attendance upon first bootstrap
+but also allows experienced users to further configure it as they wish. The
+[source code](https://github.com/TheTorProject/Lepidopter), and the building
+scripts of Lepidopter image are under a free/libre source software; GPL version
+3 license which can be reviewed and modified by anyone.
+
+## Why to use it
```
Comment (agrabeli) [inline content/post/lepidopter.md]: I would rephrase this with: "Why use Lepidopter"

File: content/post/lepidopter.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+This wouldn't be possible without the help of numerous volunteers that are
+contributed by running consistent ooni-probe network measurements tests from
+different vantage points. Up to now mostly technical savvy users needed to
+maintain their own probes and deal with system configuration, software upgrades
+and renewal of measurement test and relevant test content related to the
+person's country. Lepidopter makes it easier for people that have a Raspberry
+Pi and an SD card to be part of the OONI community and help us reveal, publish
+and document possible instances of network interference worldwide.
+
+## How to obtain it
```
Comment (agrabeli) [inline content/post/lepidopter.md]: Perhaps rephrase this with: "How to obtain Lepidopter"

File: content/post/lepidopter.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+to a Raspbeery Pi.
+
+## I don't have a Raspberry Pi how can I help?
+
+There are a number of ways that you can help with if you don't own a Raspberry
+Pi or another device were you can deploy a probe and submit dedicated reports.
+
+In many cases we are considering providing a Raspberry Pi device to interested
+parties, please let us know if you would like to be part of the OONI operators
+community and provide regular measurements.
```
Comment (agrabeli) [inline content/post/lepidopter.md]: Perhaps link "OONI operators community" to https://ooni.torproject.org/get-involved/partnership-program/ ?

File: content/post/lepidopter.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+
+
+## Keep in touch
+
+We have recently created a mailing list for future and current probe operators
```
Comment (agrabeli) [inline content/post/lepidopter.md]: Link "mailing list" to https://lists.torproject.org/cgi-bin/mailman/listinfo/ooni-dev.

Comment (agrabeli) [inline content/post/lepidopter.md]: Ah sorry, just noticed that you have included this below. :)

File: content/post/lepidopter.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+to get it touch with us and receive regular updates regarding Lepidopter and
+other probe specific operations. You can subscribe to the mailing list by
+registering
+[here](https://lists.torproject.org/cgi-bin/mailman/listinfo/ooni-operators).
+
+Find out about our regular community meetings, other mailing lists and how you
+can get involved [here](/get-involved/).
+
+Thank you!
```
Comment (agrabeli) [inline content/post/lepidopter.md]: I would remove "thank you". :)

Comment (agrabeli) [inline content/post/lepidopter.md]: Thanks, great job @anadahz! :)

**Comment by anadahz** [comment]
@agrabeli thanks a lot for the review, I have updated the post with your comments.
I haven't found a reference link for OONI research publications, perhaps let's add it to the post when we have collected this information on a link?

**Comment by agrabeli** [comment]
@anadahz since we don't currently have all research publications which use OONI data published somewhere that we can link to (yet), I'd suggest removing the sentence which mentions such publications because (1) having it without a reference is not the best thing in the world and most post views will likely happen during the first few days following the publication (2) I don't think reference to this adds any significant value to the blog about lepidopter.

**Comment by anadahz** [comment]
I have updated the post based on comments of @agrabeli.

File: content/post/lepidopter.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+[source code](https://github.com/TheTorProject/Lepidopter), and the building
+scripts of the Lepidopter image are free and open source software.
+
+## Why use Lepidopter
+
+Since 2012 OONI has collected numerous network measurements reports that show
+evidence of internet censorship, network tampering and surveillance. OONI
+measurement data have been used in a number of technical
+[reports](https://ooni.torproject.org/post/) that show how ISPs in countries
+filter and censor the internet as well as reports that include sings of network
```
Comment (hellais) [inline content/post/lepidopter.md]: s/sings/signs/

File: content/post/lepidopter.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+irregularities and tampering. This wouldn't be possible without the help of
+numerous volunteers that are contributed by running consistent ooni-probe
```
Comment (hellais) [inline content/post/lepidopter.md]: "that are contributed"

"that are contributing"

File: content/post/lepidopter.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+network measurements tests from different vantage points. Up to now mostly
+technical savvy users needed to maintain their own probes and deal with system
```
Comment (hellais) [inline content/post/lepidopter.md]: "technical savvy"

"technically savvy"

File: content/post/lepidopter.md [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+configuration, software upgrades and renewal of measurement test and relevant
+test content related to the person's country. Lepidopter makes it easier for
```
Comment (hellais) [inline content/post/lepidopter.md]: I think this whole sentence should be reworded as:

"Up to now users needed to be technically savvy to maintain their own probes, dealing with system configuration, software and measurement specific updates."

File: content/post/lepidopter.md [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+people that have a Raspberry Pi and an SD card to be part of the OONI community
+and help us reveal, publish and document possible instances of network
+interference worldwide.
+
+## How to obtain Lepidopter
+
+Please refer to the [installation instructions]
+(https://ooni.torproject.org/install/lepidopter/) where you can find a detailed
```
Comment (hellais) [inline content/post/lepidopter.md]: I think this link is broken.

File: content/post/lepidopter.md [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
+guide on how you can download, verify and install the latest Lepidopter image
+to a Raspberry Pi.
```
Comment (hellais) [inline content/post/lepidopter.md]: "to a Raspberry Pi"

"on a Raspberry Pi"

File: content/post/lepidopter.md [hunk 20]
```diff
@@ -0,0 +1,68 @@
...
+
+## I don't have a Raspberry Pi how can I help?
+
+There are a number of ways that you can help with if you don't own a Raspberry
+Pi or another device were you can deploy a probe and submit dedicated reports.
```
Comment (hellais) [inline content/post/lepidopter.md]: Probably rephrase as:

"Even if you don't have a Raspberry Pi there are still a number of ways in which you can help"

Though you don't actually list how they can help so I would perhaps drop this section entirely or put some content in it.

File: content/post/lepidopter.md [hunk 21]
```diff
@@ -0,0 +1,68 @@
...
+
+In many cases we are considering providing a Raspberry Pi device to interested
+parties, ease let us know if you would like to be part of the OONI operators
+[community](https://ooni.torproject.org/get-involved/partnership-program/) and
+provide regular measurements.
```
Comment (hellais) [inline content/post/lepidopter.md]: This should be removed since the partnership program is not about giving away free raspberry pi's to people interested in running measurements.

We should not be advertising it within this context.

**Comment by anadahz** [comment]
@hellais thank you for reviewing the PR. I have updated the post with your proposed changes.

**Comment by anadahz** [comment]
Shall we merge this PR?

**Comment by hellais** [comment]
Should we at this point perhaps wait for the new release of ooniprobe and lepidopter to come out and write up about that including content from this?

**Comment by anadahz** [comment]
Agreed let's wait for the official release.

**Comment by hellais** [comment]
@anadahz can you ensure everything that you needed from this is included in https://github.com/TheTorProject/ooni-web/pull/63 and close this PR?

**Comment by anadahz** [comment]
> @anadahz can you ensure everything that you needed from this is included in #63 and close this PR?

@hellais Do you mean #68 ?

In any case this is a blog post and not an install guide.

**Comment by hellais** [comment]
Sorry, what I meant what #76 the web ui blog post. You should include updates regarding lepidopter in the web ui blog post as it's not ideal to publish two blog posts that are basically about the same thing.

**Comment by anadahz** [comment]
@hellais this PR si a post that explains in layman's terms what is lepidopter and why someone should use it.
Do you have a nice way or idea on how to merge both #63 and #76 that can be published in a single post that will make sense?

**Comment by hellais** [comment]
> Do you have a nice way or idea on how to merge both #63 and #76 that can be published in a single post that will make sense?

IMHO all that is in here is already covered by either #63 or #76. If you think that is not the case, then please add them to those PRs.

**Comment by hellais** [comment]
Can this be closed?

**Comment by anadahz** [comment]
The content of this PR is still relevant and not covered by others PRs.
We can instead merge it when we are satisfied with the PR.
